lib/easypost/user.rb

```diff
@@ -22,8 +22,8 @@ def self.retrieve_me
   end
 
   def self.all_api_keys
-    response = EasyPost.make_request(:get, "/api_keys", @api_key)
-    return EasyPost::Util.convert_to_easypost_object(response, api_key)
+    response = EasyPost::ApiKey.all
+    return EasyPost::Util.convert_to_easypost_object(response, @api_key)
```

Comment:
It looks like the `all` method is already going to call `convert_to_easypost_object` and we don't need it here. Instead, we should be able to just leave `EasyPost::apiKey.all` here and remove line 26 and the `response =` bit.